contributors.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title: Contributors
+---
+
+## Active PIs
+
+<ul>
+{% for p in site.data.contributors %}
+{% if p.is_active_PI == true %}
+  <li>
+    {% if p.website != null %}
+        <a href="{{ p.website }}">
+        {% assign link_open = true %}
+    {% elsif p.email != null %}
+        <a href="mailto:{{ p.email }}">
+        {% assign link_open = true %}
+    {% endif %}
+    <strong>{{ p.name }}</strong>{% if link_open %}</a>{% assign link_open = false %}{% endif %}
+    {% if p.affiliation != null %} ({{ p.affiliation }}){% endif %}
+  </li>
+{% endif %}
+{% endfor %}
+</ul>
+
+## Contributors
+
+Currently, the following people contributed very significantly to the software in the
+OSCAR project.
+Additionally, people who are financed by the [SFB-TRR 195](https://www.computeralgebra.de/sfb/) mainly work on software
+for the OSCAR project.
+
+{% assign entries = site.data.contributors | sort_natural:"name" %}
+<ul>
+{% for p in site.data.contributors %}
+  <li>
+    {% if p.website != null %}
+        <a href="{{ p.website }}">
+        {% assign link_open = true %}
+    {% elsif p.email != null %}
+        <a href="mailto:{{ p.email }}">
+        {% assign link_open = true %}
+    {% endif %}
+    <strong>{{ p.name }}</strong>{% if link_open %}</a>{% assign link_open = false %}{% endif %}
+    {% if p.affiliation != null or p.paid_by_dfg == true %}
+        (
```

Comment:
Unfortunately, doing it this way inserts a lot of whitespace. In your preview, the result is that one sees e.g. `( TU Kaiserslautern , financed by ... )` instead of `(TU Kaiserslautern, financed by ...)` (note the extra whitespace; not a major issue, but still annoying)

In newer versions of Jekyll, one can avoid that by using `{%- ... -%}` etc., see https://shopify.github.io/liquid/basics/whitespace/ -- but I am not sure if the Jekyll version used by GitHub already supports it. But I'd just try.

If that doesn't work, one either needs to put everything in a single line (not so nice) or else assembler the desired text in a variable and then insert that in one go.

Reply:
Good catch, I thought I had fixed that already... But yes, the `{%- ...-%}` works here, I already used it in other places.